presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -729,6 +729,31 @@ public void testApproximateCountDistinct()
         assertQuery("SELECT approx_distinct(to_utf8(CAST(custkey AS VARCHAR)), 0.023) FROM orders", "SELECT 1036");
     }
 
+    @Test
+    public void testTotalValuesSizeInBytes()
+    {
+        // varchar
+        assertQuery("SELECT total_size_in_bytes(comment) FROM orders", "SELECT 727364");
```

Comment:
Had to leave the constant for the CHAR type column. Since the trailing space characters are trimmed when casting to CHAR the logic is not trivial, and cannot be easily expressed with the H2 SQL.